app/controllers/case_assignments_controller.rb

```diff
@@ -0,0 +1,31 @@
+class CaseAssignmentsController < ApplicationController
+  before_action :must_be_admin
+
+  def index
+    @volunteer = User.find(params[:volunteer_id]).decorate
+  end
+
+  def create
+    volunteer = User.find(params[:volunteer_id])
+    case_assignment = volunteer.case_assignments.new(case_assignment_params)
+
+    case_assignment.save
+
+    redirect_to volunteer_case_assignments_path(volunteer)
+  end
+
+  def destroy
+    volunteer = User.find(params[:volunteer_id])
+    case_assignment = volunteer.case_assignments.find(params[:id])
+
+    case_assignment.destroy
```

Comment:
Eventually we will want to use `case_assignment.is_active` instead of `.destroy` so we can view previously-assigned volunteers but because `case_contact` has `case_id` (so case contacts never vanish) it's not needed for MVP